src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -489,39 +489,41 @@ export default class MSKTable<T> extends React.Component<MSKTableProps<T>, {}> {
     }
 
     render() {
-        return (<div>
-            <ButtonToolbar style={{marginLeft:0}} className="tableMainToolbar">
-                <If condition={this.props.showFilter === true}>
-                    <div className={`pull-right form-group has-feedback input-group-sm`} style={{ display:'inline-block', marginLeft: 5}}>
-                        <input type="text" onInput={this.handlers.filterInput} className="form-control tableSearchInput" style={{ width:200 }}  />
-                        <span className="fa fa-search form-control-feedback" aria-hidden="true"></span>
-                    </div>
-                </If>
-                <If condition={this.props.showPagination === true}>
-                    { this.buildPaginationControls('pull-left topPagination', {}, this.store.paginationStatusText) }
-                </If>
-                <If condition={this.props.showColumnVisibility === true}>
-                    <ColumnVisibilityControls
-                        className="pull-right"
-                        columnVisibility={this.store.colVisProp}
-                        onColumnToggled={this.handlers.visibilityToggle}
-                        {...this.props.columnVisibilityProps}
-                    />
-                </If>
-                <If condition={this.props.showCopyDownload === true}>
-                    <CopyDownloadControls
-                        className="pull-right"
-                        downloadData={this.getDownloadData}
-                        downloadFilename="table.csv"
-                        {...this.props.copyDownloadProps}
-                    />
-                </If>
-            </ButtonToolbar>
-            <SimpleTable
-                headers={this.store.headers}
-                rows={this.store.rows}
-            />
-
-        </div>);
+        return (
+            <div>
```

Comment:
Thanks for fixing the indentation! Looks like I am not the only one preferring it this way :)